### ISSUE TITLE
Mute audio on streaming

### ISSUE THREAD

**Issue by RubSteels** [jacobbo/WebEye#5]
I'm trying to stream a live feed of a Foscam FI9826W v2 IPCam through RTSP protocol on a WPF application on Windows 10.
The IPCam uses H264 compression for video and PCM compression for audio.

Your control is great and I was able to get it going in few minutes, but I'm only able to get video, the audio is mute.
I've tried to install various codecs hoping that it would make the audio work, but I failed.

Am I missing something? Have you ever run into this problem?

**Comment by jacobbo** [comment]
Hi there, sorry, the audio is not supported yet.

**Comment by RubSteels** [comment]
Are you planning to implement it in the future? I would love to contribute but my knowledge on this matter is quite limited.

**Comment by jacobbo** [comment]
Yes, I am; unfortunately, not sure when I will have time for this feature.
